software/lib/Definitions/TypeDefinitions.h

```diff
@@ -2,25 +2,27 @@
 #define TYPEDEFINITIONS_H
 
 //Struct for Storing Color Values
-class CRGBWW{
-  public:
-    unsigned int R, G, B, WW, CW;
-
-    //operator overloading
-    //todo put this into an c++ file
-    bool operator==(const CRGBWW &color) const{
-      return (this->R == color.R &&
-          this->G == color.G &&
-          this->B == color.B &&
-          this->CW == color.CW &&
-          this->WW == color.WW);
-    }
-};
+class CRGBWW
+{
+public:
+  unsigned int R, G, B, WW, CW;
 
+  //operator overloading
+  //todo put this into an c++ file
+  bool operator==(const CRGBWW &color) const
```

Comment:
Great Idea, should implement other operators as well

Reply:
Like... you know =